src/main/resources/application.yml

```diff
@@ -62,6 +62,15 @@ address-index-settings:
     host: addressindex-api-beta.apps.devtest.onsclofo.uk
     port: 80
     
+case-service-settings:
+  case-by-id-query-path: /cases-rm-fake/{case-id}
+  rest-client-config:
+    username: serco_cks
```

Comment:
Need to find out what the case api service default credentials will be and use here

Reply:
Updated to:
```case-service-settings:
  case-by-id-query-path: /cases/{case-id}
  rest-client-config:
    scheme: http
    host: localhost
    port: 8161```

Port and query-path are correct, but not sure what to do for the 'host'